lib/workers/repository/init/inherited.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { platform } from '../../../../test/util';
+import type { RenovateConfig } from '../../../config/types';
+import {
+  CONFIG_INHERIT_NOT_FOUND,
+  CONFIG_INHERIT_PARSE_ERROR,
+  CONFIG_VALIDATION,
+} from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { mergeInheritedConfig } from './inherited';
+
+describe('workers/repository/init/inherited', () => {
+  let config: RenovateConfig;
+
+  beforeEach(() => {
+    config = {
+      repository: 'test/repo',
+      inheritConfig: true,
+      inheritConfigRepoName: 'inherit/repo',
+      inheritConfigFileName: 'config.json',
+      inheritConfigStrict: false,
+    };
+
+    platform.getRawFile.mockClear();
```

Comment:
you probably don't need this if you use the once variant for mocking below

Reply:
Removed